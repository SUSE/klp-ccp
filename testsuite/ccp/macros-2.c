#define A()
#define B A(

void pu_f(void)
{
	B)
}
