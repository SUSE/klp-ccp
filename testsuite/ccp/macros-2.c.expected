/* klp-ccp: from testsuite/ccp/macros-2.c */
#define A()
#define B A(

void klpp_pu_f(void)
{
	B)
}
